### DISCUSSION TITLE
Integrate Godot Headless Server with Nakama

### DISCUSSION THREAD

**Discussion by portwatcher** [portwatcher/blog-comments#45] [Announcements]
https://juryquinn.com/post/technology/2023-01-04

**Comment by aliotadi** [comment]
Thank you for this amazing doc you shared with the community =D you are a life saver.
i streamed reviewing your doc in twitch, if you would like to follow: https://www.twitch.tv/alistechtails
i'm new to this so any feedback will be much appretiated.
also your comment input section is not working i wrote my comment in notepad and copy/pasted in here.

**Comment by portwatcher** [comment]
> @aliotadi
> Thank you for this amazing doc you shared with the community =D you are a life saver.
> i streamed reviewing your doc in twitch, if you would like to follow: https://www.twitch.tv/alistechtails
> i'm new to this so any feedback will be much appretiated.
> also your comment input section is not working i wrote my comment in notepad and copy/pasted in here.

Though I have been developing many kinds of applications for years, I'm also new to gamedev, even new to English writing.
So I'm glad it helps!

**Comment by addmix** [comment]
Is there anywhere to download these project files?

**Reply by portwatcher** [reply]
sorry these implementations are coupled into the game project itself and I don't want to open source it.

I'd like to open source the code as they are written in godot 3 and they are useless to me.
However I didn't organize the repos very well, 3d assets, credentials, too much trouble.

If you really need it I may consider tripping it into a minimal example.

**Reply by addmix** [reply]
If it's not too much trouble, a minimal example would help me greatly, as I am a novice with Docker and Javascript. With some of Nakama's improvements with Godot 4, I would imagine this information could be valuable for the entire community.
